### PR TITLE
chore(rethinkdb): Initialize DB

### PR DESCRIPTION
**Pull request by mattkrick**
# Description

creates a new baseline for migrations. One that doesn't include RethinkDB 🎉

### CODE REVIEW

File: packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts
Comment (rafaelromcar-parabol) [inline packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts:1]: Isn't that filename problematic? if we use dates, we might have some problems with time zones

File: scripts/flushRedis.js
Comment (rafaelromcar-parabol) [inline scripts/flushRedis.js:1]: Is this script supposed to be used in production?

**Review by rafaelromcar-parabol — COMMENTED** [review]
After this migration, would RethinkDB be needed during build process?

File: packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts
Comment (mattkrick) [reply packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts:1]: the Z at the end denotes UTC & it's generated automatically

**Comment by mattkrick** [comment]
> After this migration, would RethinkDB be needed during build process?

nope, build script has been amended, but i'll clean up the rest in the next PR

File: packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts
Comment (rafaelromcar-parabol) [reply packages/server/postgres/migrations/2024-10-21T21:21:21.212Z_init.ts:1]: Ouch, read too fast 😸 thanks!

File: scripts/flushRedis.js
Comment (rafaelromcar-parabol) [reply scripts/flushRedis.js:1]: I supposed, but just wanted to be 100% sure

**Review by rafaelromcar-parabol — APPROVED** [review]

**Comment by mattkrick** [comment]
closing in favor of #10395 so we only have 1 PR

**Pull request closed without merging**